hw/ip_templates/otp_ctrl/dv/env/seq_lib/otp_ctrl_common_vseq.sv.tpl

```diff
@@ -1,6 +1,10 @@
 // Copyright lowRISC contributors (OpenTitan project).
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
+<%
+buffered_parts = [(i, part) for (i, part) in enumerate(otp_mmap["partitions"])
```

Comment:
Do we both need the index and the partitions? Down below I think only the indices are used?